docs/modules/develop/pages/classes/controllers.adoc

```diff
@@ -0,0 +1,90 @@
+= Controllers
+
+Decidim controllers, are plain Rails controllers, that usually implement the other concepts that we have described in this section.
+
+Controller classes are located in the `app/controllers/decidim/<my_plugin>` directory, and named: `<my_event>_event.rb`.
+
+```ruby
+# frozen_string_literal: true
+
+module Decidim
+  module MyPlugin
+    class ResourceController < Decidim::MyPlugin::ApplicationController
+
+    end
+  end
+end
+```
+
+The admin controllers are following the `Decidim` conventions:
+In the below example, you will be able to see how the forms, commands, queries and permissions can be used.
+
+```ruby
+# frozen_string_literal: true
+module Decidim
+  module MyPlugin
+    module Admin
+
+      class ResourceController < Decidim::MyPlugin::Admin::ApplicationController
```

Comment:
We aren't consistency in Decidim but I'd said that the Rails way is to have the controllers on plural 
```suggestion
      class ResourcesController < Decidim::MyPlugin::Admin::ApplicationController
```